src/views/product-integrations-landing/index.tsx

```diff
@@ -1,50 +1,204 @@
+						<FacetCheckbox
+							label="Community"
+							matching={matchingCommunity}
+							isChecked={communityChecked}
+							onChange={(e) => setCommunityChecked(!communityChecked)}
+						/>
+					)}
+
+					<h5>Component</h5>
+					{sortedComponents.map((component, index) => {
+						return (
+							<FacetCheckbox
+								key={component.id}
+								label={component.plural_name}
+								matching={component.occurances}
+								isChecked={componentCheckedArray[index]}
+								onChange={(e) =>
+									setComponentCheckedArray(
+										componentCheckedArray.map((v, i) => {
+											return i === index ? !v : v
+										})
+									)
+								}
+							/>
+						)
+					})}
+
+					{atLeastOneFacetSelected && (
+						<Button
+							title="Reset filters"
+							size="small"
+							onClick={(e) => {
+								setOfficialChecked(false)
+								setPartnerChecked(false)
+								setCommunityChecked(false)
+								setComponentCheckedArray(
+									componentCheckedArray.map((v, i) => {
+										return false
+									})
+								)
+							}}
+						/>
+					)}
+				</div>
+			</div>
+		</>
+	)
+}
+
+// Custom sidebar
+const IntegrationsSearchSidebar = () => {
+	const { dialogOpen, setDialogOpen } = useMobileDrawerContext()
+	return (
+		<>
+			{/* Desktop sidebar */}
+			<div className="g-hide-with-mobile-menu">
+				<SearchFacets />
+			</div>
+			{/* Mobile drawer */}
+			<div className="g-show-with-mobile-menu">
+				<Dialog
+					isOpen={dialogOpen}
+					label="Integrations filters"
+					// eslint-disable-next-line @typescript-eslint/no-empty-function
+					onDismiss={() => {}}
+					variant="bottom"
+				>
+					<button
+						className={s.exitIcon}
+						onClick={() => setDialogOpen(false)}
+						aria-label="Cancel"
+						type="button"
+					>
+						<IconX16 />
+					</button>
+					<SearchFacets />
+				</Dialog>
+			</div>
+		</>
+	)
+}
+
 export default function ProductIntegrationsLanding({
 	product,
 	integrations,
 }: ViewProps) {
 	return (
-		<BaseLayout showFooterTopBorder>
-			<div className={s.integrationsLandingPage}>
-				<div className={s.sidebar}></div>
-				<div className={s.mainArea}>
-					<div className={s.contentWrapper}>
-						<div className={s.breadcrumbWrapper}>
-							<BreadcrumbBar
-								links={[
-									{
-										title: 'Developer',
-										url: '/',
-										isCurrentPage: false,
-									},
-									{
-										title: product.name,
-										url: `/${product.slug}`,
-										isCurrentPage: false,
-									},
-									{
-										title: 'Integrations',
-										url: `/${product.slug}/integrations`,
-										isCurrentPage: true,
-									},
-								]}
-							/>
+		<IntegrationsSearchProvider integrations={integrations}>
+			<MobileDrawerContextProvider>
+				<SidebarSidecarLayout
+					sidebarNavDataLevels={
+						[
+							// @TODO — does this need values?
+						]
+					}
+					sidecarSlot={<div>TODO: Sidecar slot</div>}
```

Comment:
```suggestion
					sidecarSlot={<></>}
```

Maybe just kick in a fragment for now; TBH I'm not sure if anything will ever go in here 🤔...

Maybe we'll use merchandising slots in the future like we do on some of our Download pages.

Reply:
p.s. This page does look better with the sidecar slot being taken up with nothing though -- it was too wide before!